pkgs/by-name/pr/prism-model-checker/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  prism-model-checker-unwrapped,
+  java,
+  symlinkJoin,
+  makeWrapper,
+  writeText,
+}:
+let
+  desktopEntry = writeText "prism-model-checker-xprism.desktop" ''
+    [Desktop Entry]
+    Type=Application
+    Icon=prism-model-checker
+    Terminal=false
+    Categories=Science;Math
+    Comment=Probabalistic Symbolic Model Checker
+    Name=XPrism
+    Version=${prism-model-checker-unwrapped.version}
+    Exec=xprism
+  '';
+in
+symlinkJoin {
+  name = "prism-model-checker-${prism-model-checker-unwrapped.version}";
+  nativeBuildInputs = [ makeWrapper ];
+  paths = [
+    prism-model-checker-unwrapped
+    java
+  ];
+
+  postBuild = ''
+    rm -rf $out/bin
+    mkdir --parents $out/bin
```

`-f` is overkill and I don't see why it's needed
```suggestion
    rm -r $out/bin/*
```